internal/httputil/headers.go

```diff
@@ -3,6 +3,11 @@ package httputil
 // AuthorizationTypePomerium is for Authorization: Pomerium JWT... headers
 const AuthorizationTypePomerium = "Pomerium"
 
+// Standard headers
+const (
+	HeaderReferrer = "Referer"
```

Comment:
I'm shocked our spellchecker didn't yell at us for this...
https://en.wikipedia.org/wiki/HTTP_referer